torch/csrc/distributed/c10d/Ops.cpp

```diff
@@ -6,21 +6,6 @@
 
 namespace c10d {
 namespace {
-std::tuple<std::vector<at::Tensor>, c10::intrusive_ptr<Work>> broadcast_(
```

Comment:
I believe removing this default implementation will break custom Python PG, like the one in PyTorch/XLA.

Reply:
The way how the Python PG works is that it relies on the underlying cc ops to call the PyProcessGroup.
Now that you remove the default implementation, and then the device registered is XLA. The PyProcessGroup will never be called.

Reply:
I will add it back, thanks